assayist/common/models/content.py

```diff
@@ -27,6 +27,28 @@ class Build(AssayistStructuredNode):
     # The source location used for this build
     source_location = RelationshipTo('.source.SourceLocation', 'BUILT_FROM', cardinality=ZeroOrOne)
 
+    @staticmethod
+    def get_stubbed_builds():
+        """
+        Return all stubbed builds.
+
+        Sadly I don't think it's possible to query neomodel on both properties and relationships
```

Comment:
You can group together `has` and `filter` to achieve this. This seems to pass your test too:
```python
Build.nodes.has(source_location=False).filter(type___ne='module')
```

Reply:
Nice. I much prefer doing it the "right way".